entityx/Event.h

```diff
@@ -102,18 +65,26 @@ class Receiver : public BaseReceiver {
  * Subscriptions are automatically removed when receivers are destroyed..
  */
 class EventManager : entityx::help::NonCopyable {
+ private:
+	// Method used to get the default callback function for an event (receive method).
+	// Directly put &Receiver::receive as the default parameter of the subscribe function doesn't work with VS12.
+   template <typename E, typename Receiver>
+   static CallbackSignature<E, Receiver> getDefaultCallbackFunction() {
```

Comment:
The naming convention I use in EntityX is for methods/functions to be `underscore_separated_lowercase`. Please fix here and elsewhere (eg. in tests).